ast2vec/model2/proxbase.py

```diff
@@ -0,0 +1,155 @@
+from collections import defaultdict, deque
+from itertools import permutations, product
+import os
+from typing import Dict, Tuple
+
+import numpy
+from scipy.sparse import coo_matrix, diags
+
+from ast2vec.coocc import Cooccurrences
+from ast2vec.token_parser import TokenParser
+from ast2vec.uast import UASTModel
+from ast2vec.model2.base import Model2Base
+
+EDGE_TYPES = ["r", "t", "rt", "R", "T", "RT"]
```

Comment:
I think it's better to put description here and main should refer to this file (so docs should be in one place)